app/src/main/java/com/mapbox/vision/examples/DemoApplication.kt

```diff
@@ -9,7 +10,9 @@ class DemoApplication : MultiDexApplication() {
 
     override fun onCreate() {
         super.onCreate()
-        FirebaseCrashlytics.getInstance().setCrashlyticsCollectionEnabled(!BuildConfig.DEBUG)
+        FirebaseApp.initializeApp(this)?.let {
+            FirebaseCrashlytics.getInstance().setCrashlyticsCollectionEnabled(!BuildConfig.DEBUG)
```

Comment:
Does this mean we won't collect debug crashes?

Reply:
This means two things:
1) If `google-services.json` files are missing we do not crash on `setCrashlyticsCollectionEnabled`
2) We do not collect crashes in our debug builds even if `google-services.json` are present.

Reply:
@yunikkk - please tell me if you're OK with such logic.

Reply:
Why not enable crashlytics for debug since we have specific app for that and crashes from debug won't mess with release ones?

Reply:
Well, because you see crash reports directly in logcat. Moreover I need to perform some extra testing with stripped / unstripped `so` files. I am not sure they actually get stripped in debug.

Reply:
No, debug is not getting stripped. 
And it's possible that debug build is crashed when device is _not_ connected to the laptop and hence no logcat is visible

Reply:
@yunikkk - implemented, please take a look.